casper/src/main/resources/ListOps.rho

```diff
@@ -175,6 +175,20 @@ new  ListOps, rs(`rho:registry:insertSigned:secp256k1`), uriOut in {
     }
   } |
 
+  contract ListOps(@"all", @list, predicate, returnCh) = {
```

Comment:
This process should have a more informative name. Something like `forall` is consistent with Scala and with mathematical vocabulary generally.